README.md

```diff
@@ -1 +1,30 @@
-# DSCI_522_Group411
\ No newline at end of file
+# Predicting Avocado Prices
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+*__Creators__: Katie Birchard, Ryan Homer, Andrea Lee*
+
+## Dataset
+
+We will be analyzing the [avocado prices dataset](https://www.kaggle.com/neuromusic/avocado-prices?fbclid=IwAR35kKP-Fz0yYZj-QqsZ6iNDSVnLBncxTOG3Cce3F5EupQTVHo85ecn7SBo) retrieved from Kaggle and compiled by the Hass Avocado Board using retail scan data. The dataset consists of approximately 18,000 records over 4 years (2015 - 2018). The dataset contains information about avocado prices, PLU numbers, types, region purchased, volume sold, bags sold, and date sold.
+
+## Research Question
+
+We will be answering the research question: **What is the strongest predictor of avocado prices?**
+
+Our goal is to find the feature that most strongly predicts the average price of avocados. A natural inferential sub-question would be to first determine if any of the features correlate with avocado prices. From our results, we can also compute a rank of features by importance.
```

Comment:
We should also look for multicollinearity among the covariates. This may help when we are building the model.

Reply:
i added this as a sub question and into the analysis